src/VisualStudio/VisualStudioDiagnosticsToolWindow/OptionPages/PerformanceLoggersPage.cs

```diff
@@ -20,7 +20,7 @@ namespace Roslyn.VisualStudio.DiagnosticsWindow.OptionsPages
     internal class PerformanceLoggersPage : AbstractOptionPage
     {
         private IGlobalOptionService _optionService;
-        private IRemoteHostClientService _remoteHostClientService;
+        private IRemoteHostClientService _remoteService;
```

Comment:
no particular reason.

Reply:
💭 I found the original name easier to avoid confusion between `IRemoteHostService` and `IRemoteHostClientService`. If the name was too long, I would prefer a rename to `_remoteHostClient`, which is shorter but preserves the disambiguation characteristic.